website/docs/docs/build/view-documentation.md

```diff
@@ -0,0 +1,85 @@
+
+<Lightbox src="/img/docs/building-a-dbt-project/testing-and-documentation/ec77c45-Screen_Shot_2018-08-14_at_6.31.56_PM.png" title="Opening the DAG mini-map"/>
+
+In this example, the `fct_subscription_transactions` model only has one direct parent. By clicking the "Expand" button in the top-right corner of the window, we can pivot the graph horizontally and view the full <Term id="data-lineage">lineage</Term> for our model. This lineage is filterable using the `--select` and `--exclude` flags, which are consistent with the semantics of [model selection syntax](/reference/node-selection/syntax). Further, you can right-click to interact with the DAG, jump to documentation, or share links to your graph visualization with your coworkers.
+
+<Lightbox src="/img/docs/building-a-dbt-project/testing-and-documentation/ac97fba-Screen_Shot_2018-08-14_at_6.35.14_PM.png" title="The full lineage for a dbt model"/>
+
+## Deploying the documentation site
+
+Effortlessly deploy documentation in dbt Explorer or dbt Docs to view your project documentation and make it available to your team to collaborate.
+
+:::caution Security
+
+The `dbt docs serve` command is only intended for local/development hosting of the documentation site. Please use one of the methods listed in the next section (or similar) to ensure that your documentation site is hosted securely!
+
+:::
+
+### dbt Explorer <Lifecycle status="team,enterprise" />
+
+
+dbt Explorer automatically updates documentation after each production or staging job run using the metadata generated after each job run. This means it always has the latest results for your project &mdash; no manual deployment needed. For details on how dbt Explorer uses metadata to automatically update documentation, refer to [Generate metadata](/docs/collaborate/explore-projects#generate-metadata). 
+
+To learn how to deploy your documentation site, see [Build and view your docs with dbt Cloud](/docs/collaborate/build-and-view-your-docs).
+
+### dbt Docs
+dbt Docs was built to make it easy to host on the web. The site is "static," meaning you don't need any "dynamic" servers to serve the docs. You can host your documentation in several ways:
+
+* Host on [Amazon S3](https://docs.aws.amazon.com/AmazonS3/latest/dev/WebsiteHosting.html) (optionally [with IP access restrictions](https://docs.aws.amazon.com/AmazonS3/latest/dev/example-bucket-policies.html#example-bucket-policies-use-case-3))
+* Publish with [Netlify](https://discourse.getdbt.com/t/publishing-dbt-docs-to-netlify/121)
+* Use your own web server like Apache/Nginx
+* If you're on a dbt Cloud Developer plan, see [Build and view your docs with dbt Cloud](/docs/collaborate/build-and-view-your-docs#dbt-docs) to learn how to deploy your documentation site.
+
+For a more seamless experience, use [dbt Explorer](/docs/collaborate/explore-projects) as the recommended documentation experience in dbt Cloud. 
```

Comment:
Might be nice if you add a bit at the end, like
"Interested in using dbt Explorer for the complete dbt documentation experience, sign up for a free trial of dbt Cloud or contact us." or something like that.